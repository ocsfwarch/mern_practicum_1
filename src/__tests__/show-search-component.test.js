import React from 'react';
import ShowSearch from '../components/show-search.component.js';
import renderer from 'react-test-renderer';

it('Renders a snapshot', () => {
  const tree = renderer.create(<ShowSearch/>).toJSON();
  expect(tree).toMatchSnapshot();
});
