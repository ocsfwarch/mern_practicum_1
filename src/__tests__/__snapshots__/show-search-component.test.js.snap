// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders a snapshot 1`] = `
<div>
  <h3>
    Search Transcript List
  </h3>
  <form
    onSubmit={[Function]}
  >
    <div
      className="form-group"
    >
      <label>
        EMail
      </label>
      <input
        autoFocus={true}
        className="form-control"
        id="email"
        name="email"
        onChange={[Function]}
        title="Enter a previous email"
        type="text"
        value=""
      />
    </div>
    <div
      className="form-group"
    >
      <input
        className="btn btn-primary"
        title="Select to perform the search"
        type="submit"
        value="Search"
      />
      Â 
      <input
        className="btn btn-primary"
        onClick={[Function]}
        title="Select to close this page"
        type="button"
        value="Close"
      />
    </div>
    <div
      className="form-group"
    >
      <label>
        Results
      </label>
      <input
        className="form-control"
        id="formError"
        name="formError"
        readOnly="readOnly"
        type="text"
        value=""
      />
    </div>
  </form>
</div>
`;
